pour/web/src/status-reading.svelte

```diff
@@ -3,14 +3,29 @@
   import { onDestroy, onMount } from "svelte";
 
   export let client: GenericService;
+
   let status: string | undefined = undefined;
   let interval: number | undefined = undefined;
   type Status = {
     status: string;
   };
+
+  const displayStates: Record<string, string[]> = {
```

Comment:
Why do we define this twice, both here and in `pour/web/src/start-button.svelte`?

Reply:
Because I forgot that I want to export from status.reading.svelte and import here. Good catch!

Reply:
Actually I just need to remove here; forgot I decided to do all in status reading component 😵